index.js

```diff
@@ -242,14 +242,14 @@ MediaRecorder.prototype = {
 /**
  * Returns `true` if the MIME type specified is one the polyfill can record.
  *
- * This polyfill supports only `audio/wav`.
+ * This polyfill supports `audio/wav` and `audio/mpeg`.
  *
  * @param {string} mimeType The mimeType to check.
  *
- * @return {boolean} `true` on `audio/wav` MIME type.
+ * @return {boolean} `true` on `audio/wav` and `audio/mpeg` MIME type.
  */
 MediaRecorder.isTypeSupported = function isTypeSupported (mimeType) {
-  return /audio\/wave?/.test(mimeType)
+  return /audio\/(wave?|mpeg)/.test(mimeType)
```

Comment:
Does Polyfill support `audio/mpeg` out of the box? It will be better to enable MP3 encoding only when the user manually changed the coder (because not everyone need MP3, but the MP3 decoder takes a lot of client-side bundle).

Reply:
the lamejs lib is loaded on demand https://github.com/ai/audio-recorder-polyfill/pull/43/files#diff-683fe31c51c17e6e3c61b9d26c0dd8eeR3

Reply:
Yeap, we need to add:

1. That we have out-of-the-box MP3 support
2. Guide with exact steps to enable MP3

Reply:
Am I right, that `mpeg-encoder.js` file is not loaded to `index.js`? The developer needs to explicitly load it. So if the developer didn’t load it, Polyfill will not have MP3 support

Reply:
exactly, just like the documented example.
so maybe the isTypeSupported function should returns something like that:
`return MediaRecorder.prototype.mimeType === mimeType`

Reply:
I like `MediaRecorder.prototype.mimeType === mimeType` way

Reply:
done